indico/modules/events/surveys/blueprint.py

```diff
@@ -46,6 +47,11 @@
                  methods=('GET', 'POST'))
 _bp.add_url_rule('/manage/surveys/<int:survey_id>/open', 'open_survey', RHOpenSurvey, methods=('POST',))
 _bp.add_url_rule('/manage/surveys/<int:survey_id>/close', 'close_survey', RHCloseSurvey, methods=('POST',))
+_bp.add_url_rule('/manage/surveys/<int:survey_id>/submissions.csv', 'export_submissions', RHExportSubmissions,
+                 methods=('POST',))
+_bp.add_url_rule('/manage/surveys/<int:survey_id>/', 'delete_submissions', RHDeleteSubmissions, methods=('DELETE',))
```

Comment:
Eh, sorry for that I wanted to do it but it seems like I had forgotten about that eventually.